README.md

```diff
@@ -25,8 +25,8 @@ The **tldr-pages** project is a collection of community-maintained help pages
 for command-line tools, that aims to be a simpler, more approachable complement
 to traditional [man pages](https://en.wikipedia.org/wiki/Man_page).
 
-Maybe you are new to the command-line world? Or just a little rusty?
-Or perhaps you can't always remember the arguments to `lsof`, or `tar`?
+Maybe you are new to the command-line world; or maybe just a little rusty;
+or perhaps you can't always remember the arguments to `lsof`, or `tar`?
```

Comment:
We may want to revert this change if we want more emphasis on the questions being asked.

Reply:
I slightly prefer the questions. With the semicolons, the text seems a bit like it's describing the reader (and therefore assuming their knowledge, or rather, the lack thereof) rather than explicitly making guesses.

Reply:
```suggestion
Maybe you're new to the command-line world? Perhaps you're just a little rusty or can't always recall the arguments for commands like `lsof`, or `tar`?
```

Thoughts on this?

* I think starting sentences with a connective is always questionable. I wouldn't go as far as to say invalid, but nice to avoid.
* Using maybe/perhaps (basically synonyms) 3 times in a bit uncomfortable to read.
* Usually a sentence (in English) should have a verb and a noun, a sentence like "Or just a little rusty?" doesn't meet this, so it doesn't seem like a valid sentence.
* Users that are new to command line or don't know what `lsof` or `tar` might not be able to infer that those are examples of commands.
* Use of apostrophes, just for the sake of reducing the word count a bit. It doesn't do much but hey I tried. ^-^'

All changes are subjective, and you can ofc pick and choose which points you agree with or not.

Reply:
I'm OK with Seth's proposal :+1:. All points of the reasoning are sensible.